hw/ip/otbn/dv/otbnsim/sim/state.py

```diff
@@ -411,6 +411,14 @@ def update_mlz_flags(self, fg: int, result: int) -> None:
         '''Update M, L, Z flags for the given result'''
         self.flags[fg] = FlagReg.mlz_for_result(self.flags[fg].C, result)
 
+    @staticmethod
+    def subtract_with_borrow(a: int, b: int, borrow_in: int) -> Tuple[int, FlagReg]:
```

Comment:
Mega-nit: Could you move this above `update_mlz_flags`, so that the add and subtract methods sit next to each other?

Reply:
meta-nit: was this a minor of mega nit?

Reply:
Well, it was mega-ly nitty, so a very small nit indeed!